src/constants/skylark.ts

```diff
@@ -9,7 +9,7 @@ export const REQUEST_HEADERS = {
 };
 export const OBJECT_LIST_TABLE = {
   columnIds: {
-    objectType: "__typename", // This is the built-in GraphQL __typename
+    objectType: "object type",
```

Comment:
this used for? is the space ok? 😅

Reply:
space is good, just used in filters really